tools/plugins/preinstall/src/main/kotlin/net/corda/cli/plugins/preinstall/CheckKafka.kt

```diff
@@ -0,0 +1,265 @@
+package net.corda.cli.plugins.preinstall
+
+import net.corda.cli.plugins.preinstall.PreInstallPlugin.PluginContext
+import net.corda.cli.plugins.preinstall.PreInstallPlugin.Kafka
+import net.corda.cli.plugins.preinstall.PreInstallPlugin.ReportEntry
+import net.corda.cli.plugins.preinstall.CheckKafka.KafkaProperties.TruststoreNotFoundException
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.common.KafkaException
+import org.apache.kafka.common.Node
+import picocli.CommandLine
+import picocli.CommandLine.Option
+import picocli.CommandLine.Parameters
+import java.util.Properties
+import java.util.concurrent.Callable
+import java.util.concurrent.ExecutionException
+
+@CommandLine.Command(name = "check-kafka", description = ["Check that Kafka is up and that the credentials work."])
+class CheckKafka : Callable<Int>, PluginContext() {
+
+    @Parameters(
+        index = "0",
+        description = ["YAML file containing the Kafka, SASL, and TLS configurations"]
+    )
+    lateinit var path: String
+
+    @Option(
+        names = ["-n", "--namespace"],
+        description = ["The namespace in which to look for both the Postgres and Kafka secrets"]
```

Comment:
I'm trying (and probably failing) to be consistent about using `PostgreSQL`, at least in external facing documentation:
```suggestion
        description = ["The namespace in which to look for both the PostgreSQL and Kafka secrets"]
```